src/components/interfaces/HMI_API.xml

```diff
@@ -4154,45 +4154,54 @@
     </function>
 
     <function name="OnPutFile" messagetype="notification" >
-    <description>
-      Notification, that comes to HMI when mobile sents file with systemFile flag
-    </description>
+      <description>
+        Notification that is sent to HMI when a mobile application uploads a file
+      </description>
 
-    <param name="offset" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
-      <description>Optional offset in bytes for resuming partial data chunks</description>
-    </param>
-    <param name="length" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
-      <description>Optional length in bytes for resuming partial data chunks</description>
-    </param>
+      <param name="offset" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>Optional offset in bytes for resuming partial data chunks</description>
+      </param>
+      <param name="length" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>Optional length in bytes for resuming partial data chunks</description>
+      </param>
 
-    <param name="fileSize" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
-      <description>Full Size of file. sends in first OnPutFile notification if file is splited
-                   into many PutFiles
-      </description>
-    </param>
+      <param name="fileSize" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>Full Size of file. sends in first OnPutFile notification if file is splited
+                    into many PutFiles
+        </description>
+      </param>
 
-    <param name="FileName" type="String" maxlength="255" mandatory="true">
-      <description>File reference name.</description>
-    </param>
+      <param name="FileName" type="String" maxlength="255" mandatory="true">
```

Comment:
TODO: Since we are doing a major release, we can probably remove this parameter. It is redundant and is not actually sent by Core, `syncFileName` is sent instead.